// cohere-ai/aws.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.AwsClientV2 = exports.AwsClient = exports.SagemakerClientV2 = exports.SagemakerClient = exports.BedrockClientV2 = exports.BedrockClient = void 0;
var BedrockClient_1 = require("./BedrockClient");
Object.defineProperty(exports, "BedrockClient", { enumerable: true, get: function () { return BedrockClient_1.BedrockClient; } });
Object.defineProperty(exports, "BedrockClientV2", { enumerable: true, get: function () { return BedrockClient_1.BedrockClientV2; } });
var SagemakerClient_1 = require("./SagemakerClient");
Object.defineProperty(exports, "SagemakerClient", { enumerable: true, get: function () { return SagemakerClient_1.SagemakerClient; } });
Object.defineProperty(exports, "SagemakerClientV2", { enumerable: true, get: function () { return SagemakerClient_1.SagemakerClientV2; } });
var AwsClient_1 = require("./AwsClient");
Object.defineProperty(exports, "AwsClient", { enumerable: true, get: function () { return AwsClient_1.AwsClient; } });
Object.defineProperty(exports, "AwsClientV2", { enumerable: true, get: function () { return AwsClient_1.AwsClientV2; } });
